declarations_site/catalog/templates/declaration.jinja

```diff
@@ -13,7 +13,7 @@
 
 {% block html_desc %}{% if meta %}{{ meta.description }}{% else %}{{ _("Д​екларації") }}{% if declaration.declaration.source == "NACP" %} {{ _("НАЗК") }}{% endif %} {{ _("про майно, доходи, витрати та зобов'язання фінансового характеру") }} {{ declaration.general.last_name }} {{ declaration.general.name }} {{ _("за") }} {{ declaration.intro.declaration_year }} {{ _("рік онлайн у відкритій базі даних проекту") }}{% endif %}{% endblock %}
 
-{% block bodyclass %}not-front {% if branding_link %}branding-page {% endif %}declaration-page {% if declaration.declaration.source == "NACP" %}declaration-page-nacp{% endif %}{% endblock %}
+{% block bodyclass %}not-front body--yc-option-b {% if branding_link %}branding-page {% endif %}declaration-page {% if declaration.declaration.source == "NACP" %}declaration-page-nacp{% endif %}{% endblock %}
```

Comment:
Initial idea was to test 4 different options, are you intentionally limiting it to option b?

Reply:
Yep, we decided to hardcode version B for now